PluginsAndFeatures/azure-toolkit-for-eclipse/com.microsoft.azuretools.azureexplorer/src/com/microsoft/azuretools/azureexplorer/views/ServiceExplorerView.java

```diff
@@ -251,6 +251,11 @@ public void listChanged(final ListChangedEvent e) {
             case add:
                 // create child tree nodes for the new nodes
                 for (Node childNode : (Collection<Node>) e.getNewItems()) {
+                    // Eclipse do no support arm, so here need to skip resource management node
+					if (childNode.getClass().getName().equals(
```

Comment:
Can we use `instance of` here?

Reply:
Cannot use instance of here, here do not have the dependency of class ResourceManagementModule